### ISSUE TITLE
Segments overlapping with Right element in Header.

### ISSUE THREAD

**Issue by garylesueur** [GeekyAnts/NativeBase#1160]
Getting an unexpected overlap from the 'Right' element. This means I can't select the second segment very easily!
Shown in the image is the inspector.

<img width="376" alt="screen shot 2017-08-13 at 10 12 44" src="https://user-images.githubusercontent.com/7880838/29248431-fddc7d66-800f-11e7-9a70-124dddfc0319.png">

Nativebase 2.3.1
React-Native 0.47.1
React 16.0.0-alpha.12

```xml
<Container>
  <Header>
    <Left />
    <Body>
      <Segment>
        <Button first active>
          <Text>Seg 1</Text>
        </Button>
        <Button last>
          <Text>Seg 2</Text>
        </Button>
      </Segment>
    </Body>
    <Right>
      <Button transparent>
        <Icon name={'ios-add'} />
      </Button>
    </Right>
  </Header>
</Container>
```

I can only run in iOS Simulator and Device at the moment sorry.

If anyone has a quick fix for me to get round this for the moment, that would be appreciated.

**Comment by garylesueur** [comment]
hmm this seems to duplicate this

https://github.com/GeekyAnts/NativeBase/issues/598

but the proposed adjustment doesn't quite work, its still slightly offset.

**Comment by ErkanSensei** [comment]
I fixed this by not wrapping Segment in a Body tag. This needs to be properly fixed though.

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.3.8](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.8)
